quodlibet/quodlibet/browsers/paned/prefs.py

```diff
@@ -18,12 +20,34 @@
 from quodlibet.qltk.x import SymbolicIconImage, MenuItem, Button
 from quodlibet.qltk import Icons
 from quodlibet.qltk.menubutton import MenuButton
-from quodlibet.qltk.ccb import ConfigCheckMenuItem, ConfigCheckButton
+from quodlibet.qltk.ccb import ConfigCheckButton
 from quodlibet.util import connect_obj, escape
 from quodlibet.compat import iteritems, iterkeys
 from .util import get_headers, save_headers
 
 
+class ColumnModes(Gtk.VBox):
+    SMALL = _("Small")
+    WIDE = _("Wide")
+    COLUMNAR = _("Columnar")
+    modes = [SMALL, WIDE, COLUMNAR]
+    buttons = []
+
+    def __init__(self):
+        super(ColumnModes, self).__init__(spacing=6)
+
+        group = None
+        for mode in self.modes:
+            group = Gtk.RadioButton(group=group, label=mode)
+            try:
+                if mode == config.gettext("browsers", "pane_mode"):
+                    group.set_active(True)
+            except NoOptionError:
+                pass
```

Comment:
Instead of using this try-except, add a default value in the quodlibet/config.py file. This is a bit nicer, and also fixes the failing test. The old pane_wide_mode entry in config.py can also be removed.